src/main/resources/liquibase/harness/compatibility/foundational/update/derby/createTable.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="as" labels="foundational">
+        <loadUpdateData file="update.csv"
```

Comment:
plz add comment to this changelog why default one doesn't work

Reply:
++